### PR TITLE
vimPlugins: added vim-jinja

### PR DESCRIPTION
**Pull request by hbunke**
very small contribution to vimplugins. please take a closer look since this is my very first commit to nixpkgs ;-). thanks.

### CODE REVIEW

**Comment by hbunke** [comment]
well, I obviously can't interpret the Travis fail correctly and will simply wait.

File: pkgs/misc/vim-plugins/default.nix [hunk 1]
```diff
@@ -439,6 +439,19 @@ rec {
     };
   };
 
+  vim-jinja = buildVimPlugin {
+    name = "vim-jinja";
```
Comment (edwtjo) [inline pkgs/misc/vim-plugins/default.nix]: please add version string suffix, if there isn't a git tag we use something like, `-2015-04-23`

File: pkgs/misc/vim-plugins/default.nix [hunk 2]
```diff
@@ -439,6 +439,19 @@ rec {
...
+    src = fetchgit {
```
Comment (edwtjo) [inline pkgs/misc/vim-plugins/default.nix]: please use fetchFromGitHub instead to fetch an archive for the rev from over the GitHub api

File: pkgs/misc/vim-plugins/default.nix [hunk 3]
```diff
@@ -439,6 +439,19 @@ rec {
...
+      url = "https://github.com/lepture/vim-jinja";
+      rev = "4412484b41";
+      sha256="1z5ya953nn7233jjjs0ay61x5hfrfddx9xz31grq52cfd0ipy335";
+    };
+   meta = {
+    homepage = "https://github.com/lepture/vim-jinja";
+    #maintainers = [ stdenv.lib.maintainers.hbunke ];
```
Comment (edwtjo) [inline pkgs/misc/vim-plugins/default.nix]: Please leave yourself in

**Comment by edwtjo** [comment]
Installs fine

**Comment by hbunke** [comment]
All done. Thanks for the hints! I've didn't seen 'fetchFromGitHub' anywhere before.

**Comment by edwtjo** [comment]
Squash all commits and force push to update the pr

**Comment by edwtjo** [comment]
Thanks and welcome aboard!

**Pull request merged**